_data/advisories.yaml

```diff
@@ -12,6 +12,7 @@ texts:
   rc: "The Swarm mode feature included in Docker Engine 1.12 is a release candidate feature and might be subject to non backward-compatible changes. Some functionality may change before the feature becomes generally available."
   docker4mac-beta: "Docker for Mac is currently in public beta. Some functionality may change before the product becomes generally available."
   docker4win-beta: "Docker for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
+  toolbox: "**Legacy desktop solution.** Docker Toolbox is for older Mac and Windows systems that do not meet the requirements of the new [Docker for Mac](/docker-for-mac/) and [Docker for Windows](/docker-for-windows/) apps. We recommend updating to the newer apps, if possible."
```

Comment:
I would maybe `s/apps/desktop applications/g`
But that gets wordy, so that's optional.